pkgs/by-name/om/omnissa-horizon-client/package.nix

```diff
@@ -145,19 +145,19 @@ stdenv.mkDerivation {
   installPhase = ''
     runHook preInstall
     mkdir -p $out/bin
-    ln -s ${vmwareFHSUserEnv "vmware-view"}/bin/vmware-view $out/bin/
-    ln -s ${vmwareFHSUserEnv "vmware-usbarbitrator"}/bin/vmware-usbarbitrator $out/bin/
+    ln -s ${omnissaFHSUserEnv "horizon-client"}/bin/horizon-client $out/bin/
+    ln -s ${omnissaFHSUserEnv "omnissa-usbarbitrator"}/bin/omnissa-usbarbitrator $out/bin/
     runHook postInstall
   '';
 
-  unwrapped = vmwareHorizonClientFiles;
+  unwrapped = omnissaHorizonClientFiles;
 
   passthru.updateScript = ./update.sh;
 
   meta = with lib; {
     inherit mainProgram;
-    description = "Allows you to connect to your VMware Horizon virtual desktop";
-    homepage = "https://www.vmware.com/go/viewclients";
+    description = "Allows you to connect to your Omnissa Horizon virtual desktop";
+    homepage = "https://www.omnissa.com/products/horizon-8/";
     license = licenses.unfree;
     platforms = [ "x86_64-linux" ];
     maintainers = [ ];
```

Comment:
Are you willing to be a maintainer?

Reply:
Yes for the time being I'll have to work with this anyway.